src/controllers/users.js

```diff
@@ -0,0 +1,23 @@
+const Users = require('../models/user.js');
+
+module.exports = {
+    async register(req, res){
+        const { user_email } = req.body;
+        try{
+            if (await Users.findOne({ user_email }))
```

Comment:
A @emysdias comentou e no momento não me atentei a questão de um email pode ser não existente. Não sei se existe cadastros que usam o email e não o verificam... pode ser feito:

- Podemos fazer verificação;
- Podemos fazer com o email institucional.

Na questão futura de um possível envio de notificação(caso a postagem seja resolvida), notícias e relatórios por email, o email terá que ser verificado.

Reply:
Acredito que a questão de validar o e-email pode ser trabalhada em uma sprint futura.

Reply:
Qual algum empecilho para ser trabalhada nessa issue?